mmpose/core/evaluation/top_down_eval.py

```diff
@@ -319,6 +385,14 @@ def keypoints_from_heatmaps(heatmaps,
         kernel (int): Gaussian kernel size (K) for modulation, which should
             match the heatmap gaussian sigma when training.
             K=17 for sigma=3 and k=11 for sigma=2.
+        kpd (float): Keypoint pose distance for UDP.
```

Comment:
`kdp` is a little cryptic, how about use its full name?

Reply:
Keypoint_pose_distance instead?

Reply:
I'm okay with long names

Reply:
ok, point taken